app.js

```diff
@@ -26,6 +26,14 @@ if (!process.env.RACKSPACE_USERNAME
   throw new Error('Required parameters not provided from the environment');
 }
 
+routes.client.createContainer({
```

Comment:
I would work this a little differently; I would have it first check if exists, and if not, then create, and finally, listen.